packages/1ui/src/components/ExploreListGrid/ExploreListGrid.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+export interface ExploreListGridProps
```

Comment:
Naming question on this @Vitalsine85 @0xjojikun -- do we want to call this something specific like `Explore_` instead of a more generic name? Only reason I'm thinking about that is that this could be more generally useful for devs outside of the specific Explore usecase

Reply:
I was actually thinking of just removing it from 1ui actually, since it is very specific to Portal. They already have ListGrid which serves the same purpose, we just needed another version for Explore because there were issues with how the grid was rendering from one page to another.

Reply:
That makes sense! That's likely the best approach if we don't really need it in 1ui and since it's Portal specific we can have it be there